pkg/server/plugin/upstreamauthority/gcpcas/gcpcas_test.go

```diff
@@ -0,0 +1,58 @@
+package gcpcas
```

Comment:
This plugin could certainly use more test coverage. I'm happy to provide guidance on this if wanted.